src/style.css

```diff
@@ -1,3 +1,144 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
The font-family is set to 'Roboto', which is a web font. Ensure that the font is correctly linked in your HTML file to avoid fallback to the default sans-serif font.